lib/topological_inventory/openshift/operations/worker.rb

```diff
@@ -40,33 +40,34 @@ def stop
         attr_accessor :messaging_client_opts, :client, :api_client
 
         def process_message(_client, msg)
-          logger.info("Processing order service with msg: #{msg.payload}")
+          logger.info("Processing #{msg.message} with msg: #{msg.payload}")
           #TODO: Move to separate module later when more message types are expected aside from just ordering
-          context = order_service(msg.payload["service_plan_id"], msg.payload["order_params"])
-          update_task(msg.payload["task_id"].to_s, context)
+          order_service(msg.payload)
         rescue => e
           logger.error(e.message)
           logger.error(e.backtrace.join("\n"))
           nil
         end
 
-        def order_service(service_plan_id, order_params)
+        def order_service(payload)
+          service_plan_id = payload["service_plan_id"].to_s
```

Comment:
This wasn't calling `.to_s` before, right? Will adding it cause any problems?

Reply:
It was in the retriever https://github.com/ManageIQ/topological_inventory-collector-openshift/pull/41/files#diff-ee6c74bbf8bf14cfc5756b3f280d5e66L9